### PR TITLE
CNV#36834: Document VM config changes in UI

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/CNV-36834

Link to docs preview:
https://71431--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/getting_started/virt-web-console-overview

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by ousleyp** [comment]
@aspauldi and I will be reviewing this PR together tomorrow at 11am.

**Comment by aspauldi** [comment]
/ label peer-review-in-progress

File: virt/getting_started/virt-web-console-overview.adoc [hunk 1]
```diff
@@ -38,6 +38,9 @@ The *Virtualization* section of the {product-title} web console contains the fol
 
 |xref:../../virt/getting_started/virt-web-console-overview.adoc#migrationpolicies-page_virt-web-console-overview[*MigrationPolicies* page]
 |Create and manage migration policies for workloads.
+
+|*Checkups* page
+|Run network latency and storage checkups for virtual machines.
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:43]: It's probably not in the scope of this PR, but this is the only page in the table without a link to more information.

File: virt/getting_started/virt-web-console-overview.adoc [hunk 2]
```diff
@@ -741,32 +695,74 @@ You configure scheduling, network interfaces, disks, and other options on the *C
...
+|Setting
+|Description
+
+|*Description*
+|Click the edit icon to enter a description.
+
+|*Workload profile*
+|Click the edit icon to edit the workload profile.
+
+|*CPU\|Memory*
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:740]:
```suggestion
|*CPU \| Memory*
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 3]
```diff
@@ -741,32 +695,74 @@ You configure scheduling, network interfaces, disks, and other options on the *C
...
+|Click the edit icon to edit the CPU\|Memory request. Restart the virtual machine to apply the change.
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:741]:
```suggestion
|Click the edit icon to edit the *CPU \| Memory* request. Restart the virtual machine to apply the change.
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 4]
```diff
@@ -786,20 +782,20 @@ You manage disks on the *Disks* tab.
...
-|*File systems* table
-|List of virtual machine file systems.
+|*Add Config Map, Secret or Service Account*
+|Click the link and select a config map, secret, or service account from the resource list.
 |====
 =====
 
-[id="virtualmachine-details-network-interfaces_virt-web-console-overview"]
+[id="virtualmachine-details-network_virt-web-console-overview"]
 ===== Network interfaces tab
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:796]:
```suggestion
===== Network tab
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 5]
```diff
@@ -786,20 +782,20 @@ You manage disks on the *Disks* tab.
...
 
-You manage network interfaces on the *Network interfaces* tab.
+You manage network interfaces on the *Network* tab.
 
 .*Network interfaces* tab
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:800]:
```suggestion
.*Network interfaces* table
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 6]
```diff
@@ -898,18 +897,29 @@ Restart the virtual machine to apply changes.
...
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
+
+|*Labels*
+|Click the edit icon to manage your labels.
 
 |*Sysprep*
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:921]: Is Sysprep just a typo? 

```suggestion
|*Annotations*
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 7]
```diff
@@ -1010,6 +1020,9 @@ You view the status conditions and volume snapshot status on the *Diagnostics* t
 
 |*Volume snapshot status* table
 |List of volumes, their snapshot enablement status, and reason.
+
+|*DataVolume stats* table
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:1024]:
```suggestion
|*DataVolume status* table
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 8]
```diff
@@ -1010,6 +1020,9 @@ You view the status conditions and volume snapshot status on the *Diagnostics* t
...
+|List of DataVolumes, their phase, and progress.
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:1025]:
```suggestion
|List of data volumes and their *Phase* and *Progress* values.
```

File: virt/getting_started/virt-web-console-overview.adoc [hunk 9]
```diff
@@ -1634,10 +1638,10 @@ You configure the persistent volume claim (PVC) of the bootable volume by editin
 |Description
 
 |*Name*
-|PVC name.
+|DataSource name.
```
Comment (aspauldi) [inline virt/getting_started/virt-web-console-overview.adoc:1641]: For every general reference to an API object (data source, instance type, virtual machine), the formatting should follow these API guidelines: 

"[API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references)

For terms that are API objects, the way they are written depends on whether the term is a general reference or an actual reference to the object.

General references

A general reference is any time you are speaking conceptually, or generally, about these components in a cluster.

When referring to API object terms in general usage, use lowercase and separate multi-word API objects. Default to following this guidance unless you are specifically interacting with/referring to the API object (see [Object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references))."

This comment applies to each API object instance in the table descriptions.

**Review by aspauldi — COMMENTED** [review]
Looks really good, Daniel! What a lot of work. Pan and I worked on this together, so let us know if you have questions. 

Thanks!

File: virt/getting_started/virt-web-console-overview.adoc [hunk 10]
```diff
@@ -38,6 +38,9 @@ The *Virtualization* section of the {product-title} web console contains the fol
 
 |xref:../../virt/getting_started/virt-web-console-overview.adoc#migrationpolicies-page_virt-web-console-overview[*MigrationPolicies* page]
 |Create and manage migration policies for workloads.
+
+|*Checkups* page
+|Run network latency and storage checkups for virtual machines.
 |====
```
[the remark this answers is not in the document]
Comment (danielclowers) [reply virt/getting_started/virt-web-console-overview.adoc:44]: i felt like this probably needed to be added and i just overlooked it. thanks for catching it! i've added a Checkups section with table to address this

File: virt/getting_started/virt-web-console-overview.adoc (hunk 6 above)
Comment (ousleyp) [reply virt/getting_started/virt-web-console-overview.adoc:933]: In this particular section, the two UI elements are *Labels* and *Annotations*, so we thought it was safe to assume that "Sysprep" was a typo left over from the previous table :) It should be changed to *Annotations* here.

Comment (danielclowers) [reply virt/getting_started/virt-web-console-overview.adoc:933]: ohhh good catch!

File: virt/getting_started/virt-web-console-overview.adoc [hunk 11]
```diff
@@ -1025,6 +1035,9 @@ You view the status conditions and volume snapshot status on the *Diagnostics* t
 
 |*Volume snapshot status* table
 |List of volumes, their snapshot enablement status, and reason.
+
+|*DataVolume status* table
+|List of data volumes, their *Phase*, and *Progress* values.
```
Comment (ousleyp) [inline virt/getting_started/virt-web-console-overview.adoc:1040]:
```suggestion
|List of data volumes and their *Phase* and *Progress* values.
```

File: virt/getting_started/virt-web-console-overview.adoc (hunk 9 above)
Comment (ousleyp) [reply virt/getting_started/virt-web-console-overview.adoc:1656]: Hey @danielclowers this comment is still unresolved; in the descriptions where you're using general references to API objects, they should be lowercase and have spaces (data source, instance type, etc.)

**Review by ousleyp — COMMENTED** [review]
Just a few things to finish up, but it's looking good!

File: virt/getting_started/virt-web-console-overview.adoc (hunk 9 above)
Comment (danielclowers) [reply virt/getting_started/virt-web-console-overview.adoc:1656]: overlooked this. fixing!

**Review by ousleyp — COMMENTED** [review]
A few more nits, sorry! Happy to merge after these are fixed. Thank you for doing so much awesome work to improve the accuracy of this page!

**Pull request merged**